openwisp-config/Makefile

```diff
@@ -10,40 +10,29 @@ PKG_VERSION:=$(shell cat ../VERSION)
 PKG_RELEASE:=1
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(BUILD_VARIANT)/$(PKG_SOURCE_SUBDIR)
 PKG_LICENSE:=GPL-3.0
+PKG_MAINTAINER:=Federico Capoano <f.capoano@openwisp.io>
 
 include $(INCLUDE_DIR)/package.mk
 
-define Package/openwisp-config/default
-	TITLE:=Remote configuration management agent ($(2) variant)
+define Package/openwisp-config
+	TITLE:=Remote configuration management agent
 	CATEGORY:=Administration
 	SECTION:=admin
 	SUBMENU:=openwisp
-	DEPENDS:=+curl +lua +libuci-lua +luafilesystem +luci-lib-nixio $(3)
-	VARIANT:=$(1)
+	DEPENDS:=+curl +lua +libuci-lua +luafilesystem +luci-lib-nixio +ca-certificates
+	PROVIDES:= openwisp-config-openssl openwisp-config-mbedtls openwisp-config-wolfssl openwisp-config-nossl
```

Comment:
One stupid question
If we are removing definitions of these packages, then why they are mentioned here?

Reply:
I added this because there may be someone who has entered this as a dependency somewhere. But we can also delete it, I would even prefer that.

Reply:
@devkapilbansal Should I remove it, would only be for back value compatibility if someone refers to one of the packages that are now no longer available

Reply:
@feckert the upcoming release of openwisp will be contain a number of backward incompatible changes, so if we want to do this it may be good to take the chance and do it now, from then onwards we can refrain from doing so many other backward incompatible changes like we have been needing to do lately, what do you think?

Reply:
@nemesisdesign thansk for you feedback.
Should fit now